projects.php

```diff
@@ -0,0 +1,43 @@
+<?php
+require_once './database.php';
+
+//get projects
+$sql = "SELECT * FROM `project`;";
+$query = $db->prepare($sql);
+$query->execute();
+$projects = $query->fetchAll();
+
+//get tags
+$sql = "SELECT * FROM `tag`;";
+$query = $db->prepare($sql);
+$query->execute();
+$tags = $query->fetchAll();
+
+//get project tag relationships
+$sql = "SELECT *, `tag`.`name` as name FROM `project_tag`
+        JOIN `tag` ON `tag_id`=`tag`.`id`;";
+$query = $db->prepare($sql);
+$query->execute();
+$project_tags = $query->fetchAll();
+
+
+foreach($projects as $project) {
+    echo '<div class="';
+    //alternate classes so the card style switches every other project
+    if ($project['id'] % 2 == 0) { 
```

Comment:
Fine as long as the projects you're going to display are sequential/fit the pattern odd, even, odd etc.

Reply:
Suspect this might not be the case in the future

Reply:
Agreed, I'll probably need to refactor to a better solution. It's offically added to my roadmap!